docs/integrator/data-flow.md

```
@@ -444,29 +447,25 @@ values := input.GetValuesForComponent("gpu-operator")
-# Helm values.yaml - receives values map
-driver:
-  version: {{ index .Values "driver.version" }}
-  
-# README.md - receives combined map with Values + Script
-Driver Version: {{ index .Values "driver.version" }}
-Namespace: {{ .Script.Namespace }}
+**Template usage:** per-component `values.yaml` is the component's values map
+marshaled to YAML directly (not a Go-templated file). `README.md` and
+`deploy.sh` are rendered from `readmeTemplateData` / `deployTemplateData`,
+which expose fields like `RecipeVersion`, `Components`, and `Constraints`:
+```gotemplate
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)